src/WebUI/WorkLoads/DeploymentsTable.tsx

```diff
@@ -258,14 +259,16 @@ export class DeploymentsTable extends BaseComponent<IDeploymentsTableProperties,
     private static _getHeadingContent(deployment: V1Deployment): JSX.Element {
         return (
             <div>
-                <h3>{deployment.metadata.name}</h3>
-                <div className="kube-flex-row">
-                    <span className="secondary-text kind-tag"> {Resources.DeploymentText} </span>
-                    <LabelGroup labelProps={Utils.getUILabelModelArray(deployment.metadata.labels)}
-                        wrappingBehavior={WrappingBehavior.OneLine}
-                        fadeOutOverflow={true}>
-                    </LabelGroup>
-                </div>
+                <span className="kube-flex-row">
+                    <h3>{deployment.metadata.name}</h3>
+                    <div className="heading-labels">
+                        <LabelGroup labelProps={Utils.getUILabelModelArray(deployment.metadata.labels)}
+                            wrappingBehavior={WrappingBehavior.OneLine}
+                            fadeOutOverflow={true}>
+                        </LabelGroup>
+                    </div>
+                </span>
+                <span className="secondary-text kind-tag"> {Resources.DeploymentText} </span>
```

Comment:
remove spaces in the text
<span className="secondary-text kind-tag"> {Resources.DeploymentText} </span>